IDE/Espressif/ESP-IDF/examples/wolfssl_test/components/wolfssl/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+                message(STATUS "")
+                #
+                # Note we already checked above and confirmed there's not another wolfSSL installed in the ESP-IDF components.
+                #
+                # We won't do anything else here, as it will be assumed the original install completed successfully.
+                #
+            else()
+                #
+                # This is the developer repo mode. wolfSSL will be assume to be not installed to ESP-IDF nor local project
+                # In this configuration, we are likely running a wolfSSL example found directly in the repo.
+                #
+                message(STATUS "")
+                message(STATUS "Using developer repo ./components/wolfssl in CMAKE_HOME_DIRECTORY = $ENV{CMAKE_HOME_DIRECTORY}")
+                message(STATUS "")
+
+                message(STATUS "************************************************************************************************")
+                # When in developer mode, we are typically running wolfSSL examples such as benchmark or test directories.
+                # However, the as-cloned or distributed wolfSSL does not have the ./include/ directory, so we'll add it as needed.
+                #
+                # first check if there's a [root]/include/user_settings.h
+                if( EXISTS "${WOLFSSL_ROOT}/include/user_settings.h" )
+                    message(STATUS "Found wolfSSL user_settings.h in ${WOLFSSL_ROOT}/include/user_settings.h")
+                else()
+                    message(STATUS "Installing wolfSSL user_settings.h to ${WOLFSSL_ROOT}/include/user_settings.h")
+                    file(COPY "${WOLFSSL_ROOT}/IDE/Espressif/ESP-IDF/user_settings.h" DESTINATION "${WOLFSSL_ROOT}/include/")
+                endif() # user_settings.h
+
+                # next check if there's a [root]/include/config.h
+                if( EXISTS "${WOLFSSL_ROOT}/include/config.h" )
+                    message(STATUS "Found wolfSSL config.h in ${WOLFSSL_ROOT}/include/config.h")
+                else()
+                    message(STATUS "Installing wolfSSL config.h to ${WOLFSSL_ROOT}/include/config.h")
+                    file(COPY "${WOLFSSL_ROOT}/IDE/Espressif/ESP-IDF/dummy_config_h" DESTINATION "${WOLFSSL_ROOT}/include/")
+                    file(RENAME "${WOLFSSL_ROOT}/include/dummy_config_h" "${WOLFSSL_ROOT}/include/config.h")
+                endif() # config.h
+                message(STATUS "************************************************************************************************")
+                message(STATUS "")
+            endif()
+
+        else()
+            # we did not find a ./components/wolfssl/include/ directory froim this pass of cmake.
+            if($WOLFSSL_FOUND_IDF)
+                message(STATUS "")
+                message(STATUS "WARNING: wolfSSL not found.")
+                message(STATUS "")
+            else()
```

Comment:
Remove unnecessary 'else' block

Reply:
Good catch! Actually I was missing a message, so the `else()` stayed and I added a [message](https://github.com/gojimmypi/wolfssl/blob/50cd511cf28ea83497b66c56a2969c0d5a51f646/IDE/Espressif/ESP-IDF/examples/wolfssl_test/components/wolfssl/CMakeLists.txt#L140) regarding path.